### DISCUSSION TITLE
Is turborepo viable for backend applications using Node and Typescript?

### DISCUSSION THREAD

**Discussion by antoinegag** [vercel/turborepo#149] [Help] [answered]
Most of the documentation seems to be geared toward frontend applications using say Next or React. 

Wondering if turborepo would fit the case where you have for example a NextJS/React app and a dedicated backend API project in the same repo. Would such project benefit from using Turborepo?

Sorry if the answer seems obvious, I'm not too familiar with build systems, cheers!

**Comment by jaredpalmer** [comment, marked as answer]
Absolutely! You can use Turborepo with anything you can fit into a yarn/npm/pnpm workspace

**Reply by antoinegag** [reply]
Awesome! Thank you!

**Comment by thesohailjafri** [comment]
I think you should don't go with turborepo for backend_frontend development. you wont find docs for it deployment. I have somehow managed to refactor code with npm workspace and deployed the backend on render but it took 2 all nigher to figure out the deployment

**Reply by dubey2k** [reply]
Where did you find the documentation?
